test/libsolidity/semanticTests/functionTypes/selector_assignment_expression_3.sol

```diff
@@ -0,0 +1,14 @@
+contract D {
+    function f() external {}
+}
+
+contract C {
+    bool public z;
+    function h() public {
+        ((z = true) ? D.f : D.f).selector;
```

Comment:
This is the original case, isn't it? It will fail after we merge the other PR, so you can remove it now that we have the other one.